test/cppsuite/configs/hs_cleanup_default.txt

```diff
@@ -19,22 +19,25 @@ runtime_monitor=
         max=10,
         max=100000000,
         postrun=true,
+        save=true,
     ),
     cc_pages_removed=
     (
         max=10000000,
         postrun=true,
+        save=true,
     ),
     stat_cache_size=
     (
-        max=1000,
+        max=100,
```

Comment:
Is this change intentional? It seems unrelated to stat reporting

Reply:
It is intentional but I am happy to revert it. I am pretty sure it is a mistake that I spotted when I was reviewing the stats.

Reply:
No worries then, just wanted to confirm it wasn't accidental